core/src/urls.py

```diff
@@ -13,9 +13,10 @@
     1. Import the include() function: from django.conf.urls import url, include
     2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
-from django.conf.urls import url
+from django.conf.urls import include, url
 from django.contrib import admin
 
 urlpatterns = [
+	url(r'^videos/', include('videos.urls')),
```

Comment:
Please fix the indentation error. Commonly, we use space instead of tab (not like Java). And each tab equals to 4 spaces.